SUMMARY.md

```diff
@@ -0,0 +1,25 @@
+## Table of Contents
+
+* [Connecting to Database](content/ContectingToDatabase/ConnectingToDatabase.q.md) ** - minutes**
+    > Learn how you can run PostgreSQL database operations in PHP using a specific class or command to establish a connection.
```

Comment:
I think we can replace the word `run` here with __`perform`__. Please check. Thanks.

Or we can rephrase the statement to:

> Learn how you can use a specific class or command to establish a connection between `PHP` and `PostgreSQL`.